guides/tutorials/authenticate_with_service_account.md

```diff
@@ -2,6 +2,8 @@
 
 This tutorial provides step-by-step instructions for authenticating your MongoDB Atlas CLI profile using Service Accounts, allowing you to automate and and manage MongoDB resources. Service Accounts, also referred to as OAuth Applications, allows programmatic access through a secure Client ID and Secret—ideal for scripting use cases and CI/CD workflows.
 
+__Note:__ This process is interactive. For programmatic authentication, set enviroment variables `MONGODB_ATLAS_*` before using the AtlasCLI. See [Atlas CLI Environment Variables](https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-env-variables/) for all supported environment variables.
+
 ---
```

Comment:
I like this approach because it keeps the command example simple - perhaps we should list the environment variables users will need to set here though?